src/snowflake/snowpark/modin/plugin/_internal/get_dummies_utils.py

```diff
@@ -0,0 +1,311 @@
+#
+# Copyright (c) 2012-2024 Snowflake Computing Inc. All rights reserved.
+#
+
+from collections.abc import Hashable
+
+from snowflake.snowpark.functions import (
+    col,
+    min as min_,
+)
+
+from snowflake.snowpark.modin.plugin._internal.ordered_dataframe import OrderingColumn
+from snowflake.snowpark.modin.plugin._internal.utils import (
+    pandas_lit,
+    extract_pandas_label_from_snowflake_quoted_identifier,
+)
+from snowflake.snowpark.modin.plugin._internal import (
+    join_utils,
+)
+
+from snowflake.snowpark.modin.plugin._internal.frame import InternalFrame
+from snowflake.snowpark.modin.plugin.utils.error_message import ErrorMessage
+
+
+ROW_POSITION_INDEX_COLUMN_PANDAS_LABEL = "row_position_index"
+DUMMY_COLUMN_PANDAS_LABEL = "lit_one"
```

Comment:
qq: what does `lit_one` mean?  should it be
`DUMMY_COLUMN_PANDAS_LABEL = "dummy"`
according to same convention as the ROW_POSITION_INDEX_COLUMN_PANDAS_LABEL?

Reply:
In our codebase, `lit` is typically an abbreviation of "literal". So `lit_one` here indicates that the column only contains the value 1, which seems descriptive enough to me.

Reply:
I see. Got it. So that's in line with `panda_lit(1)`

Reply:
right, I guess I can rename the DUMMY_COLUMN_PANDAS_LABLE to lit one column label to be more clear.